src/main/java/com/softserve/ita/java442/cityDonut/exception/CustomExceptionHandler.java

```diff
@@ -40,6 +41,18 @@ public final ResponseEntity notFoundException(WebRequest request){
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    @ExceptionHandler(MalformedURLException.class)
+    public final ResponseEntity malformedURLException(WebRequest request){
+        ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
+    }
+
```

Comment:
47, 53 rows. Are u sure that HttpStatus should be Not_Found?

Reply:
Which one do you propose instead of NOT_FOUND?

Reply:
BAD_REQUEST